config/mini_games/speedrun.lua

```diff
@@ -0,0 +1,115 @@
+--- Each research, entity, or item will count once per entry towards the speed run progress
+return {
+    { --- Steel Axe
+        goal = 'Complete the research for steel axe.',
+        name = 'Steel Axe',
+        research = {
+            'steel-axe','steel-processing'
+        },
+        items = {
+            'automation-science-pack','copper-plate','copper-ore','iron-gear-wheel',
+            'iron-plate','iron-ore','electronic-circuit','copper-cable','transport-belt'
+        },
+        entities = {
+            'steam-engine', 'lab'
+        }
+    },
+    { --- Getting on Track
+        goal = 'Craft and place a locomotive on rails.',
+        name = 'Getting on Track',
+        research = {
+            'railway','logistics-2','logistics','logistic-science-pack','engine','steel-processing'
+        },
+        items = {
+            'locomotive','automation-science-pack','logistic-science-pack','steel-plate','iron-plate',
+            'iron-ore','electronic-circuit','copper-cable','copper-plate','copper-ore',
+            'engine-unit','iron-gear-wheel','pipe','transport-belt','inserter'
+        },
+        entities = {
+            'steam-engine', 'lab', 'assembling-machine-1', 'assembling-machine-2', 'locomotive'
+        }
+    },
+    { --- Reduced
+        goal = 'Craft one set of power armour MK2.',
+        name = 'Reduced',
```

Comment:
why is this called reduced

Reply:
Its called Reduced because it is a reduced version of standard, reduce meaning some stuff removed.